### PR TITLE
Fix message bullets and email notification tests

### PR DESCRIPTION
**Pull request by saturninoabril**
#### Summary
Fix message bullets and email notification tests

#### Ticket Link
none, failing tests on daily Cypress tests

### CODE REVIEW

File: e2e/cypress/integration/channel/message_bullets_spec.js [hunk 1]
```diff
@@ -33,7 +33,7 @@ describe('Message', () => {
                 const expectedSize = '13.5px';
 
                 cy.get('p').first().should('have.text', 'This is a normal sentence.').and('have.css', 'font-size', expectedSize);
-                cy.get('ol li').first().should('have.text', 'this is point 1\nthis is a bullet under 1\n').and('have.css', 'font-size', expectedSize);
+                cy.get('ol li').first().should('have.text', 'this is point 1\nthis is a bullet under 1').and('have.css', 'font-size', expectedSize);
```
Comment (saturninoabril) [inline e2e/cypress/integration/channel/message_bullets_spec.js]: This is recent structure change on rendering markdown.

File: e2e/cypress/integration/email/mention_email_notification_spec.js [hunk 2]
```diff
@@ -56,10 +56,11 @@ describe('Email notification', () => {
             const bodyText = response.data.body.text.split('\n');
 
             const permalink = bodyText[9].match(reUrl)[0];
-            const permalinkPostId = permalink.split('/')[5];
+            const permalinkPostId = permalink.split('/')[6];
 
-            // # Visit permalink (e.g. click on email link)
+            // # Visit permalink (e.g. click on email link), view in browser to proceed
             cy.visit(permalink);
+            cy.findByText('View in Browser').click();
```
Comment (saturninoabril) [inline e2e/cypress/integration/email/mention_email_notification_spec.js]: This is due to the new feature - Landing Page.

**Comment by srkgupta** [comment]
Hi @saturninoabril 
The test mention_email_notification_spec fails on my localhost and cypress test instance because it looks like the email contents have changed. Can you please take a look at it and fix it as well. 

Screenshot 2020-01-22 at 9 46 34 PM

I have attached the bodyText for your reference:
Screenshot 2020-01-22 at 9 46 11 PM

**Comment by saturninoabril** [comment]
@srkgupta The newly created user is not member of a team (`ad-1`) where it got mentioned. That's why it's not getting email notification on mention.  Fix by specifically visiting such team, so that it will be added on creation.  Thanks for checking.

@prapti Feel free to review again, thanks

**Pull request merged**